src/pages/Main/ItemList/Best.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import BestItemList from './BestItemList';
+import './best.scss';
+
+const Best = ({ bestRef }) => {
+  const [bestCount, setBestCount] = useState(1);
+  const [bestList, setBestList] = useState([]);
+  const [buttonActive, setButtonActive] = useState(false);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/bestData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setBestList(data);
+      });
```

Comment:
- http://localhost:3000/ 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다. 
- fetch함수의 get method는 method의 default 값입니다. 따라서 생략이 가능합니다. 
아래 코드 참고해주세요:)
```suggestion
    fetch('/data/bestData.json')
      .then(res => res.json())
      .then(data => {
        setBestList(data);
      });
```